src/main/java/seedu/internbuddy/logic/commands/WithdrawCommand.java

```diff
@@ -70,7 +72,13 @@ public CommandResult execute(Model model) throws CommandException {
                 newStatus, applicationList, companyToEdit.getIsFavourite());
 
         model.setCompany(companyToEdit, editedCompany);
-        return new CommandResult(String.format(MESSAGE_WITHDRAW_APPLICATION_SUCCESS, applicationToWithdraw));
+        String successMessage = String.format(MESSAGE_WITHDRAW_APPLICATION_SUCCESS,
+                Messages.format(applicationToWithdraw), companyToEdit.getName());
+        if (applicationList.isEmpty()) {
+            return new CommandResult(successMessage + "\nNo more applications for "
+                    + companyToEdit.getName().fullName + ". Status has been set to CLOSED.");
+        }
+        return new CommandResult(successMessage);
```

Comment:
Love the new feedback message provided to our users, prompting them about the lack of active applications in the target company after the current command execution ends, thus changing the status to CLOSED.